frontend/src/components/DetailsTable.test.tsx

```diff
@@ -55,60 +101,502 @@ describe('DetailsTable', () => {
         ]}
       />,
     );
-    expect(tree).toMatchSnapshot();
+    expect(container).toMatchInlineSnapshot(`
+      <div>
+        <div>
+          <div
+            class="row"
+          >
+            <span
+              class="key"
+            >
+              key
+            </span>
+            <span
+              class="valueText"
+            >
+              Lorem Ipsum is simply dummy text of the printing and typesetting
+            industry. Lorem Ipsum has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a galley of type and
+            scrambled it to make a type specimen book. It has survived not only five
+            centuries, but also the leap into electronic typesetting, remaining
+            essentially unchanged. It was popularised in the 1960s with the release
+            of Letraset sheets containing Lorem Ipsum passages, and more recently
+            with desktop publishing software like Aldus PageMaker including versions
+            of Lorem Ipsum.
+            </span>
+          </div>
+        </div>
+      </div>
+    `);
   });
 
   it('shows key and value in row', () => {
-    const tree = shallow(<DetailsTable fields={[['key', 'value']]} />);
-    expect(tree).toMatchSnapshot();
+    const { container } = render(<DetailsTable fields={[['key', 'value']]} />);
+    expect(container).toMatchInlineSnapshot(`
+      <div>
+        <div>
+          <div
+            class="row"
+          >
+            <span
+              class="key"
+            >
+              key
+            </span>
+            <span
+              class="valueText"
+            >
+              value
+            </span>
+          </div>
+        </div>
+      </div>
+    `);
   });
 
   it('shows key and JSON value in row', () => {
-    const tree = shallow(
+    const { container } = render(
       <DetailsTable fields={[['key', JSON.stringify([{ jsonKey: 'jsonValue' }])]]} />,
     );
-    expect(tree).toMatchSnapshot();
+    expect(container).toMatchInlineSnapshot(`
+      <div>
+        <div>
+          <div
+            class="row"
+          >
+            <span
+              class="key"
+            >
+              key
+            </span>
+            <div
```

Comment:
nit: snapshot for ace editor is long and it didn't even include the actual content.
Recommend mocking it out like https://github.com/Bobgy/pipelines/blob/68c55e035640be29f89b8f4896d045acf6595c91/frontend/src/components/PodYaml.test.tsx#L7

Reply:
ok done